src/react-app-env.d.ts

```diff
@@ -5,6 +5,9 @@ import { AsyncAPIDocument } from '@asyncapi/parser';
 
 declare global {
   interface Window {
+    // needed by monaco YAML plugin
+    monaco: typeof monacoAPI;
```

Comment:
`monaco` and `Monaco`? Can't we make it just one? Maybe make Studio get `monaco` instead?